.github/workflows/test.yml

```diff
@@ -71,35 +74,35 @@ jobs:
         uses: "./.github/actions/smoke_tests"
 
       - name: Create MRT credentials file
-        if: env.IS_NOT_FORK == 'true' && env.IS_LATEST_NPM == 'true' && ( env.DEVELOP == 'true' || env.RELEASE == 'true' || env.V3 == 'true' )
+        if: env.IS_NOT_FORK == 'true' && env.IS_MRT_NODE == 'true' && ( env.DEVELOP == 'true' || env.RELEASE == 'true' || env.V3 == 'true' )
         uses: "./.github/actions/create_mrt"
         with:
           mobify_user: ${{ secrets.MOBIFY_CLIENT_USER }}
           mobify_api_key: ${{ secrets.MOBIFY_CLIENT_API_KEY }}
 
       - name: Push Bundle to MRT (Development)
-        if: env.IS_NOT_FORK == 'true' && env.IS_LATEST_NPM == 'true' && env.DEVELOP == 'true'
+        if: env.IS_NOT_FORK == 'true' && env.IS_MRT_NODE == 'true' && env.DEVELOP == 'true'
```

Comment:
Replacing with `IS_MRT_NODE` makes more sense because in this case:
- first of all, what matters here is the _Node_ version (not npm)
- we're pushing a bundle to MRT, and MRT supports Node 16
- while PWA Kit supports Node 16 and also 18

`IS_LATEST_NPM` originally made sense because the latest NPM used in PWA Kit (and CI too) at the time was for Node 16. But now it is no longer true because we're supporting Node 18 too.